File: test_app/test_app/customization/todo/todo.py
import frappe
from test_app.test_app.customization.todo.doc_event.actual_code_file import validate_custom_code
from frappe.desk.doctype.todo.todo import ToDo

def validate(doc, method=None):
	validate_custom_code(doc)

class CustomToDo(ToDo):
	def validate(self):
		frappe.msgprint("Massage From Override Class Method")

File: test_app/test_app/customization/todo/todo.js
frappe.ui.form.on("ToDo", {
	validate(frm) {
		console.log(" Trigger function FROM apps/test_app/test_app/test_app/customization/todo/todo.js");
		test_app.todo.todo.demo(frm.doc.priority)
	}
});

